### DISCUSSION TITLE
I have my own domain, what is the cheapest way to deploy this docker project?

### DISCUSSION THREAD

**Discussion by Root-FTW** [AstraSurge/gpteams#31] [Q&A] [answered]
I have my own domain, what is the cheapest way to deploy this docker project?

**Comment by suikodev** [comment, marked as answer]
I remember promising you that I would add support for one-click deployment on railway, I want to know if this is the cheapest way for you.
If I don't work overtime today, I will research on how to deploy on railway tonight (UTC+8).

**Reply by Root-FTW** [reply]
> I remember promising you that I would add support for one-click deployment on railway, I want to know if this is the cheapest way for you. If I don't work overtime today, I will research on how to deploy on railway tonight (UTC+8).

currently I use railway.app, the developer version, and it is more than enough for this kind of projects :)
